common/thorhelper/thorxmlwrite.cpp

```diff
@@ -645,7 +645,6 @@ void CommonJsonWriter::outputBeginNested(const char *fieldname, bool nestChildre
     if (!*fieldname && !checkUnamedArrayItem(true))
         return;
 
-    flush(false);
```

Comment:
@richardkchapman These were overkill compared to the XML implementation, so this makes JSON behavior more like XML.  The issue when using extend  meant that the buffers from multiple extend outputs might be interleaved when they shouldn't have been.